### DISCUSSION TITLE
Feature Request: Search Query Performance - ASIN VIEW

### DISCUSSION THREAD

**Discussion by samborambo305** [amzn/selling-partner-api-models#575] [Ideas]
- Make the Search Query Performance data available via the SP API with the ASIN VIEW

**Comment by sungolivia** [comment]
Hey @samborambo305 thanks for this feature request. We actually have this feature request already submitted and will follow up when there is an update :)

**Reply by amistry007** [reply]
@samborambo305 do you have a link to the feature request that we can bookmark so as to know the status of the progress of this?

**Reply by james-tempus** [reply]
Bumping again for where this is submitted, I only found this request

**Comment by tspicer** [comment]
@sungolivia  any update on this report type? There are a number of dubious vendors offering these reports via screen scraping bots. Those of us who only use the official API are left in the cold.

**Comment by HeGuanyuan** [comment]
@sungolivia 

I would like to express my strong support for this feature request. I believe this feature is crucial for us as sellers because it would significantly enhance our ability to analyze ASIN search performance. Currently, the process of manually downloading this data is not only time-consuming but also inefficient. The implementation of this feature would greatly streamline our workflow, enabling us to make faster and more accurate business decisions. I hope the development team will prioritize the development of this feature and implement it swiftly to help us better compete in the challenging marketplace.

Thank you for your efforts and contributions.

**Reply by tspicer** [reply]
Agreed @HeGuanyuan. It has been over a year since the last update on this that the feature was submitted. It is important to have parity between the user interface reports and the API. Not having this creates significant drift.

**Comment by jakeleventhal** [comment]


**Reply by james-tempus** [reply]
I don't think we're looking for workarounds here, this should be available through the API.
